libs/portals/admin/regulations-admin/src/utils/routing.ts

```diff
@@ -0,0 +1,59 @@
+import { DraftImpactId, RegulationDraftId } from '@island.is/regulations/admin'
+import { generatePath } from 'react-router-dom'
+import { isUuid } from 'uuidv4'
+import { RegulationsAdminPaths } from '../lib/paths'
+import { Step } from '../types'
+
+const { RegulationsAdminEditStep, RegulationsAdminEdit } = RegulationsAdminPaths
+
+// ---------------------------------------------------------------------------
+
+type GetEditUrlFn = {
+  (stepName: Step): string
+  (draftId: RegulationDraftId, stepName?: Step): string
+}
+
+export const getEditUrl: GetEditUrlFn = (
+  draftIdOrStepName: RegulationDraftId | Step,
+  stepName?: Step,
+) => {
+  if (isUuid(draftIdOrStepName)) {
+    const path = stepName ? RegulationsAdminEditStep : RegulationsAdminEdit
+    return generatePath(path, {
+      draftId: draftIdOrStepName,
+      stepName: stepName ?? null,
+    })
+  }
+  return draftIdOrStepName || '.'
+}
+
+// ---------------------------------------------------------------------------
+
+export const getHomeUrl = () => RegulationsAdminPaths.RegulationsAdminRoot
+
+// ---------------------------------------------------------------------------
+
+/* // TODO: implement urls to impacts?
+export const getImpactUrl = (
+  impactId: DraftImpactId,
+  draftId?: RegulationDraftId,
+) =>
+  (draftId ? generatePath(RegulationsAdminEdit, { draftId }) : '') +
+  `impacts?impact=${encodeURI(impactId)}`
+*/
```

Comment:
Todo!

Reply:
Þetta var hugmynd til að geta linkað beint á impact skráningu en ég efa að það sé þörf á þessu í dag. Þetta meikar ekki mikið sens í þessu ferli.

Reply:
I'll remove it!